formatter-providers-plugin/README.md

```diff
@@ -0,0 +1,9 @@
+## formatter-providers-plugin
+
+Simple Theia plugin to demonstrate formatting abilities of Languages API.
+
+It shows how to register formatting providers and provides simple examples of formatters.
+
+### Demo
+
+[![Formatters Demo](http://img.youtube.com/vi/YoYd_nh7Uzo/0.jpg)](https://www.youtube.com/watch?v=YoYd_nh7Uzo)
```

Comment:
I am not sure about links to personal accounts. I would prefer a gif or a few gifs hosted on github

Reply:
I mean a few seconds gifs which demonstrate essentials.

Reply:
good suggestion, I'll add some gifs instead.